tardis/apps/publication_workflow/components/utils/PublicationToast.jsx

```diff
@@ -0,0 +1,35 @@
+import { Toast } from 'react-bootstrap';
+import * as PropTypes from 'prop-types';
+import React from 'react';
+
+const PublicationToast = ({ onClose, show, toastMessage }) => (
+  <Toast
+    onClose={onClose}
+    show={show}
+    delay={5000}
+    animation
+    style={{
+      position: 'absolute', top: window.pageYOffset, right: 10, zIndex: 9999,
+    }}
+    autohide
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 8 spaces but found 4.](https://app.codacy.com/gh/mytardis/mytardis/pullRequest?prid=7933947)
```suggestion
        autohide
```